easybuild/easyconfigs/p/PyTorch-Ignite/PyTorch-Ignite-0.4.9-foss-2021a-CUDA-11.3.1.eb

```
@@ -0,0 +1,33 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-Ignite'
+version = '0.4.9'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://pytorch-ignite.ai/'
+description = """
+Ignite is a high-level library to help with training and evaluating neural
+networks in PyTorch flexibly and transparently."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', True),
+    ('Python', '3.9.5'),
+    ('PyTorch', '1.10.0', versionsuffix),
+]
+
+exts_list = [
+    ('pytorch-lightning', version, {
```

surely this should be
```suggestion
    ('pytorch-ignite', version, {
```

---

whoops :)